extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -19,7 +19,7 @@ void FuzzyDuck::BeginFuzzing() {
 		seed = random_engine.NextRandomInteger();
 	}
 	if (max_queries == 0) {
-		max_queries = NumericLimits<idx_t>::Maximum();
+		throw BinderException("Provide a max_queries argument greater than 0");
```

Comment:
IIRC this is intentional - calling without providing a number of max queries just means "fuzz until cancelled manually or an error is discovered. I'm fine with making this explicit though.

In general it doesn't really matter - nobody aside from us (developers) should be using the fuzzing infrastructure.